src/App/Command/EnableCommand.php

```diff
@@ -65,10 +70,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         fwrite($handle, '];' . "\n");
         fclose($handle);
 
-        if (!empty($enabledPackages)) {
-            $io->success("Enabled packages:\n — " . implode("\n — ", $enabledPackages));
-        } else {
+        if (empty($alreadyEnabledPackages) && empty($enabledPackages)) {
             $io->info('Packages not found.');
```

Comment:
Similar to https://github.com/yiisoft/yii-dev-tool/pull/219/files#r947529868

Reply:
Done.